builtin/bisect--helper.c

```diff
@@ -419,6 +471,11 @@ int cmd_bisect__helper(int argc, const char **argv, const char *prefix)
 		set_terms(&terms, argv[1], argv[0]);
 		res = bisect_next_check(&terms, argc == 3 ? argv[2] : NULL);
 		break;
+	case BISECT_TERMS:
+		if (argc > 1)
+			return error(_("--bisect-terms requires 0 or 1 argument"));
+		res = bisect_terms(&terms, argc == 1 ? argv[0] : NULL);
+		break;
 	default:
 		return error("BUG: unknown subcommand '%d'", cmdmode);
 	}
```

Comment:
I think the changes to `git-bisect.sh` (https://github.com/gitgitgadget/git/commit/db7b281c8ceb8d0662b4dc35edcd3422965b2500#diff-712d3a389eb3a42550cf1f6fa8c2e754R355) were dropped from this commit by accident...